E/Elemental/build_tarballs.jl

```diff
@@ -0,0 +1,64 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Elemental"
+version = v"0.87.7"
+
+# Collection of sources required to build Elemental
+sources = [
+    GitSource("https://github.com/elemental/Elemental.git",
+              "477e503a7a840cc1a75173552711b980505a0b06"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+if [[ $nbits == 64 ]]; then
```

Comment:
On aarch64 our openblas doesn't use LP64, see https://github.com/JuliaPackaging/Yggdrasil/blob/40b71c9995cd0c6c95f20e930d4847c281e804d7/O/OpenBLAS/common.jl#L45
```suggestion
if [[ $nbits == 64 ]] && [[ "${target}" != aarch64-* ]]; then
```